widget/table.go

```diff
@@ -344,6 +344,17 @@ func (t *Table) TouchCancel(*mobile.TouchEvent) {
 // Implements: fyne.Focusable
 func (t *Table) TypedKey(event *fyne.KeyEvent) {
 	switch event.Name {
+	case fyne.KeyHome:
+		t.ScrollToTop()
+		t.Refresh()
```

Comment:
The Refresh() calls here should not be necessary since the Scroll APIs already refresh

Reply:
You were partially right.

The default ScrollToTop/ScrollToBottom did not work correctly, a t.content.Refresh() was missing before t.finishScroll(), so I added it. Without it, it does scroll to top or bottom, but the table becomes blank.

Reply:
Sorry this has sat for so long. Something doesn't seem right though. Are you saying that the current "ScrollToTop" or "ScrollToBottom" don't work unless there is a Refresh call in the developers code?